src/components/ActivityCard.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-container pa-0>
+    <v-card>
+      <v-img
+        :height="imageHeight"
+        :src="imgSrc"
+      ></v-img>
+      <v-card-title class="no-padding-bottom">
+        <div class="text-truncate">
+          <p class="title text-truncate">
+            {{ title }}
+          </p>
+          <p class="body-2 grey--text text-truncate">
+            {{ description }}
+          </p>
+        </div>
+      </v-card-title>
+      <v-card-actions class="no-padding-top">
+        <v-btn flat>
+          Saber mais
+        </v-btn>
+        <v-spacer></v-spacer>
+        <v-btn :color="buttonColor">
+          {{ finalized ? 'Finalizada' : 'Feito' }}
```

Comment:
We shouldn't use any logic in the template section, so we can use a computed property called "buttonLabel", for example. By the way, what is the difference between "Finalizada" and "Feito" and what is the link with the boolean finalized